component/pom.xml

```diff
@@ -585,6 +585,7 @@
                             io.siddhi.query.*;version="${siddhi.import.version.range}",
                             *;resolution:=optional
                         </Import-Package>
+                        <DynamicImport-Package>*</DynamicImport-Package>
```

Comment:
Avoid using dynamic imports. Add the necessary packages in the import-packages section

Reply:
fixed in ee838f87eb853dd4cccb33250c2fa8f7d620a3cc